rsconf/component/devbox.py

```diff
@@ -121,14 +121,20 @@ def _gen_secrets(self, jc):
         s = super().gen_identity_and_host_ssh_keys(jc, "host", encrypt_identity=True)
         self.secrets = PKDict({k: s[k] for k in ("host_key_f", "identity_pub_f")})
 
-    def _jupyter_bashrc(self, z, path):
+    def _jupyter_bashrc(self, jc, z, path):
         self.install_access(mode="600")
         self.install_ensure_file_exists(path)
         for n in ("package_path", "sim_types"):
-            if n in z:
-                self._env(f"SIREPO_FEATURE_CONFIG_{n.upper()}", ":".join(z[n]), path)
-        z.service_port = z.ssh_port + z.ssh_service_port_difference
-        z.job_supervisor_port = z.service_port + z.ssh_service_port_difference
+            if n in jc[self.module_name]:
+                self._env(
+                    f"SIREPO_FEATURE_CONFIG_{n.upper()}",
+                    ":".join(jc[self.module_name][n]),
+                    path,
+                )
+        z.service_port = z.ssh_port + jc[self.module_name].ssh_service_port_difference
```

Comment:
I think `delta` is better or maybe `inc`.

Reply:
This is already configuration so to change would be backwards incompatible. Worth it? 

https://github.com/radiasoft/rsconf/pull/432/files#diff-b972143365867703eab8e161882070dc75a75c0547d7f5744cc8281cd7dc7fa8L130

Reply:
Yes, it's used minimally, and can be added before the release without a conflict.

I think it should be defaulted in any event. A reasonable default is 1000.

Reply:
ef32892